packages/menu/src/Menu/Menu.types.ts

```diff
@@ -45,4 +45,9 @@ export interface MenuProps extends Omit<PopoverProps, 'active'> {
    * default: `false`
    */
   darkMode?: boolean;
+
+  /**
+   * id passed to the menu dropdown.
+   */
+  id?: string;
```

Comment:
Should MenuProps extend the HTMLElement props of the root element (i.e. the `div` inside the Popover) instead? It means our `...rest` on the `ul` might end up being more verbose, but it feels like it would allow the developer more flexibility and for us to not duplicate HTML element props like `id`

Reply:
Would it be ok if we're extending a `div` but spreading it on a `ul`? I guess the props would actually be the same.

Reply:
Will you file that ticket please